docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+  --num-nodes "1" \
+  --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --enable-autoupgrade --enable-autorepair --no-enable-basic-auth \
+  --network "projects/${PROJECT_ID}/global/networks/${NETWORK_NAME}" \
+  --subnetwork "projects/${PROJECT_ID}/regions/us-central1/subnetworks/${SUBNETWORK_NAME}" \
+  --default-max-pods-per-node "110" \
+  --addons HorizontalPodAutoscaling,HttpLoadBalancing
+```
+
+This may take a while to finish.
+
+4. Connect to your GKE cluster
+```bash
+gcloud container clusters get-credentials "my-existing-cluster" --zone us-central1-a --project "${PROJECT_ID}"
+```
+
+5. Make sure `kubectl` is able to communicate with your GKE cluster
+```bash
+kubectl cluster-info
+```
+
+### Crossplane
+Using your GKE cluster:
+
+* Install Crossplane from alpha channel using the [Crossplane Installation
+  Guide](../install-crossplane.md#master)
+* Install the GCP stack into Crossplane using the [GCP stack
+  section](../install-crossplane.md#gcp-stack) of the install guide.
+
+We will be using `gcp-infra-dev` namespace to store our cloud-specific resource
+classes, cloud resources and providers. But you can use any namespace name for
+that matter.
+```bash
+kubectl create namespace gcp-infra-dev
+export INFRA_NAMESPACE=gcp-infra-dev
+```
+
+#### GCP Provider
+It is essential to make sure that a GCP service account to be used by 
+Crossplane is configured with all the necessary permissions outlined in the 
+[provider guide](../cloud-providers/gcp/gcp-provider.md).
+
+Using the service account json `crossplane-gcp-provider-key.json`:
+* Generate Base64 encoded value to store in a `Secret`:
+```bash
+export BASE64ENCODED_GCP_PROVIDER_CREDS=$(base64 crossplane-gcp-provider-key.json | tr -d "\n")
+```
+
+* Define a GCP `Provider` and `Secret`:
+```bash
+cat > gcp-provider.yaml <<EOF
+---
+apiVersion: v1
+data:
+  credentials.json: $BASE64ENCODED_GCP_PROVIDER_CREDS
+kind: Secret
+metadata:
+  namespace: $INFRA_NAMESPACE
+  name: gcp-provider-creds
+type: Opaque
+---
+apiVersion: gcp.crossplane.io/v1alpha2
+kind: Provider
+metadata:
+  namespace: $INFRA_NAMESPACE
+  name: gcp-provider
+spec:
+  credentialsSecretRef:
+    name: gcp-provider-creds
+    key: credentials.json
+  projectID: $PROJECT_ID
+EOF
+
+kubectl apply -f gcp-provider.yaml
+
+```
+* Verify GCP provider was successfully registered by the crossplane
+```bash
+kubectl get providers.gcp.crossplane.io -n $INFRA_NAMESPACE
+kubectl get secrets -n $INFRA_NAMESPACE
+```
+
+#### Cloud-Specific Resource Classes
+Cloud-specific resource classes are used to define a reusable configuration for
+a specific managed service. Wordpress requires a MySQL database, which can be
+satisfied by a [Google Cloud SQL Instance](https://cloud.google.com/sql/docs/mysql/) instance.
```

Comment:
`Google Cloud SQL Instance instance` should be more like `Google Cloud SQL Instance`